### ISSUE TITLE
🐛 [Bug]  No page scrolling and gray page overlay when creating a post

### ISSUE THREAD

**Issue by hermann-san** [Ocelot-Social-Community/Ocelot-Social#4691]
## :bug: Bug Report

When I write a post and add a link to the post, often times the windows is blocking me from being able to scroll the page or or down, after I've added a link. The whole page gets a transparent gray overlay and only reloading the page helps to get out of that mode. I can still write and click on links, but I can't scroll anymore. Pressing the ESC key on the keyboard doesn't help.

I can't reproduce this very well. It just appears to happen randomly. Sometimes after a while of editing a post. Sometimes I only add a picture to the post and the title but nothing in the body of the post. It has happened on my todays post.

This is Firefox 92.0.1 on Linux Mint.

**Comment by Tirokk** [comment]
Thanks for reporting this issue!
I never had that experience @hermann-san .

**Comment by hermann-san** [comment]
@Tirokk   David also had this issue every now and then. He said, a workaround might be the zoom out of the page (Browser function) in order to be able to see the save button, in the case it's not visible.

**Comment by Tirokk** [comment]
I don’t understand what you want to say @hermann-san …

**Comment by sushidave** [comment]
Could be related to the toolbar legend feature #3673 .
@hermann-san If the overlay happens again please try the following: Click on the question mark button. Does it make the overlay disappear?

**Comment by hermann-san** [comment]
@Tirokk   in the below example I've written my post, but I can't save the post, because there is no save button visible and since scrolling is not working, I can't get there either. The workaround is to use the browser zoom function  (keyboard shortcut ctlr and +-). Once I made the page very small, I can also see the save button.

@sushidave I've tested the formatting legend button,  but it didn't lead to the bug. When I encounter it again, I will try your hint. 
In general I would agree that it seem to be related to that button.
Thanks.

![grafik](https://user-images.githubusercontent.com/7679654/135844868-afafd122-e641-455b-8765-b6b2f11a37fb.png)
